src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -0,0 +1,1024 @@
+            km_l2_m2 = exp(kslope_l2_m2 * t_soi24_degC_col(c,j) + kint_l2_m2) * kmod_l2_m2
+            km_s3_m1 = exp(kslope_s3_m1 * t_soi24_degC_col(c,j) + kint_s3_m1) * kmod_s3_m1 / p_scalar(c,j)
+            km_s3_m2 = exp(kslope_s3_m2 * t_soi24_degC_col(c,j) + kint_s3_m2) * kmod_s3_m2 / p_scalar(c,j)
+
+            ! Product of w_scalar * depth_scalar * o_scalar
+            w_d_o_scalars = w_scalar(c,j) * depth_scalar(c,j) * o_scalar(c,j)
+
+            ! decomp_k used in SoilBiogeochemPotentialMod.F90
+            ! TODO Defined correctly here?
+            ! TODO Combine multiple terms of same donor pools...
+            ! TODO Where to calculate m1_pool and m2_pool?
+            ! TODO When to introduce litter/soil biomass to the calculation?
+            ! TODO Excluded spinup terms for now. See cwd below how used.
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_pool(c,j) / &
+                                       (km_l1_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m2 * m2_pool(c,j) / &
+                                       (km_l1_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m1 * m1_pool(c,j) / &
+                                       (km_l2_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m2 * m2_pool(c,j) / &
+                                       (km_l2_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m1 * m1_pool(c,j) / &
+                                       (km_s3_m1 + m1_pool(c,j))) * &
+                                      w_d_o_scalars
+            decomp_k(c,j,i_pas_som) = (vmax_s3_m2 * m2_pool(c,j) / &
+                                       (km_s3_m2 + m2_pool(c,j))) * &
+                                      w_d_o_scalars
+
+            decomp_k(c,j,i_act_som) = k_s1_s3 * w_d_o_scalars
+            decomp_k(c,j,i_slo_som) = k_s2_s3 * w_d_o_scalars
+
+            decomp_k(c,j,i_cop_mic) = tau_m1_s1 * m1_pool(c,j)**(densdep) * &
+                                      fphys(c) * w_d_o_scalars
+            decomp_k(c,j,i_cop_mic) = tau_m1_s2 * m1_pool(c,j)**(densdep) * &
+                                      fchem(c) * w_d_o_scalars
+            decomp_k(c,j,i_cop_mic) = tau_m1_s3 * m1_pool(c,j)**(densdep) * &
+                                      faval(c) *w_d_o_scalars
+            decomp_k(c,j,i_oli_mic) = tau_m2_s1 * m2_pool(c,j)**(densdep) * &
+                                      fphys(c) * w_d_o_scalars
+            decomp_k(c,j,i_oli_mic) = tau_m2_s2 * m2_pool(c,j)**(densdep) * &
+                                      fchem(c) * w_d_o_scalars
+            decomp_k(c,j,i_oli_mic) = tau_m2_s3 * m2_pool(c,j)**(densdep) * &
```

Comment:
in this case sum the three to get decomp_k for i_cop_mic
similarly for i_oli_mic